api/src/main/java/org/openmrs/module/sync/api/db/hibernate/HibernateSyncDAO.java

```diff
@@ -605,7 +605,7 @@ public boolean setFlushModeManual() throws DAOException {
 		
 		// don't need to set it if its already manual
 		if (!isManualAlready)
-			sessionFactory.getCurrentSession().setFlushMode(org.hibernate.FlushMode.MANUAL);
+			sessionFactory.getCurrentSession().setHibernateFlushMode(org.hibernate.FlushMode.MANUAL);

```

Comment:
The Hibernate API changed

Reply:
Do you plan to use these changes with openmrs platform versions below 2.4?

Reply:
No, I plan to support 2 branches of code and not try to support them all from one.